### ISSUE TITLE
Material Design icons not loading

### ISSUE THREAD

**Issue by MaksimSavich** [siduck/chadwm#123]
I have installed everything necessary to properly run chadwm, or so I think. The icons aren't loading and after much troubleshooting I am unsure why. Below I have linked some images of what I am experiencing.

![chadwm1](https://user-images.githubusercontent.com/51101287/205355028-f687be5d-aeab-4dc8-badf-c7f7f83484c9.PNG)
![chadwm2](https://user-images.githubusercontent.com/51101287/205355041-a47f016d-f679-4366-9292-82588bdead0d.PNG)

**Comment by MaksimSavich** [comment]
**TLDR**; Solved the issue by reinstalling my VM and then the fonts and icons. 

Turns out I was using the normal JetBrains Mono fonts which I believe are the same but the naming is different; that didn't seem to be an issue. I believe the issue originates from an initial DWM install where I likely specified persistent configs to the vanilla DWM which was causing my VM to be confused. TBH I am not 100% sure what could have been the issue, but a good ole reset did the trick.

![chadwm3_working](https://user-images.githubusercontent.com/51101287/205415313-6b92c160-500f-405e-8c69-a7390719c647.PNG)
